cmd/bosun/sched/sched.go

```diff
@@ -49,10 +52,30 @@ type Schedule struct {
 	db            *bolt.DB
 }
 
-func (s *Schedule) TimeLock(t miniprofiler.Timer) {
-	t.Step("lock", func(t miniprofiler.Timer) {
-		s.Lock()
-	})
+func init() {
+	metadata.AddMetricMeta(
+		"bosun.schedule.lock_time", metadata.Counter, metadata.MilliSecond,
+		"The length of time spent waiting for, or holding the schedule lock.")
```

Comment:
Can we omit "The" at the beginning of these sentences? Most of the other descs I see don't have it? And remove the comma. Or add another one after "holding".